config.analysis

```diff
@@ -131,6 +134,18 @@ compare_with_obs = True
 # output, N_movavg=12 corresponds to a 12-month moving average window)
 N_movavg = 12
 
+
+[moc_postprocess]
+filename  = ../mpaso.hist.am.timeSeriesStats.yr100-119.avg.nc
+filenamemesh = ../mpaso.rst.0002-01-01_00000.nc
+filenamemask = ../EC60to30v3_SingleRegionAtlanticWTransportTransects_masks.nc
```

Comment:
I would comment out the Atlantic mask file for now, because we want to be plotting only the global MOC for this PR (see my initial comment and point 1)).